packages/jest-runtime/src/index.ts

```diff
@@ -83,8 +83,7 @@ const defaultTransformOptions: InternalModuleOptions = {
   supportsStaticESM: false,
 };
 
-type InitialModule = Partial<Module> &
-  Pick<Module, 'children' | 'exports' | 'filename' | 'id' | 'loaded'>;
+type InitialModule = Omit<Module, 'require' | 'parent' | 'paths'>;
```

Comment:
might consider doing `new Module` and inspect what's on it to see what fields we should include, but meh